charts/redhat/redhat/redhat-dotnet-imagestreams/0.0.2/src/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+description: |-
+  Build and run .NET applications on UBI.
+  For more information about using this builder image, including OpenShift considerations,
+  see https://github.com/redhat-developer/s2i-dotnetcore.
+annotations:
+  charts.openshift.io/name: Red Hat .NET imagestreams.
+  charts.openshift.io/provider: Red Hat
+  charts.openshift.io/providerType: redhat
+apiVersion: v2
+appVersion: "9.0"
+kubeVersion: '>=1.20.0'
```

Comment:
Looking at https://access.redhat.com/solutions/4870701 and https://access.redhat.com/support/policy/updates/openshift, it looks like we can even go up to kube version 1.25.

Reply:
Because we're not using introducing anything that needs a newer version of Kubernetes, we can keep our version requirement at the same value as the 0.0.1 version.